.changelog/22418.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
```

Comment:
this can be `:bug` instead of `enhancement`

Reply:
Thanks @drewmullen, changes done.